src/components/Project/strings.ts

```diff
@@ -0,0 +1,9 @@
+import { createLocalizedString } from 'basics/Locale';
+
+const str = {
+  executionsTotal: (n: number) => `${n} Executions`,
+  tasksTotal: (n: number) => `${n} Tasks`,
+};
+
+export { patternKey } from 'basics/Locale';
```

Comment:
Nice, we plan to move to string files, to simplify localization in future, when/if we would be ready for it :)